packages/desktop-client/src/components/reports/reports/Spending.tsx

```diff
@@ -54,18 +54,23 @@ export function Spending() {
   const navigate = useNavigate();
   const { isNarrowWidth } = useResponsive();
 
-  if (!data) {
```

Comment:
I'm not sure why this was moved down. By leaving it in it's original place it alleviates the need to put `data &&` into any const variables or logic expressions below this point.

Reply:
This was a mistake. Thank you for pointing it out :)